openstudiocore/src/model/ThermalZone.cpp

```diff
@@ -2376,6 +2386,13 @@ namespace detail {
       tz.setZoneControlContaminantController(controllerClone);
     }
 
+    // Note: Even though rendering color is a children, ParentObject::clone isn't called by HVACComponent::clone
+    // So we need to explicitly clone the rendering color
+    if( auto t_color = renderingColor() ) {
```

Comment:
@jmarrec is ThermalZone the only color-able object that has a custom clone?

Reply:
Yes, because `ThermalZone::clone` is the only one that never calls `ParentObject_Impl::clone`